README.md

```diff
@@ -16,7 +16,7 @@
 -->
 
 # Hudi
-Apache Hudi (Incubating) (pronounced Hoodie) stands for `Hadoop Upserts anD Incrementals`. 
+Apache Hudi (Incubating) (pronounced Hoodie) stands for `Hadoop Upsert Delete and Incremental`. 
```

Comment:
nit:  `Upserts Deletes and Incrementals`?

Reply:
> nit: `Upserts Deletes and Incrementals`?

Paste from whitepaper
>In short, Hudi (Hadoop Upsert Delete and Incremental) is an analytical, scan-optimized data storage abstraction which enables applying mutations to data in HDFS on the order of few minutes and chaining of incremental processing

I think `Hadoop Upsert Delete and Incremental` is ok. @vinothchandar

Reply:
But grammatically the plural is better.. let me merge this and fix on top/